jarbas/core/querysets.py

```diff
@@ -29,6 +29,9 @@ def list_distinct(self, field, order_by_field, query=None):
         self = self.values(field, order_by_field).order_by(order_by_field)
         return self.distinct()
 
+    def receipt(self):
+        return self.exclude(receipt_fetched=False)
```

Comment:
This is not the best way to do it. Jarbas could have tried to fetch the receipt (receipt_fetched = True) but still the URL wasn't available (receipt_url = None). I would use `self.exclude(receipt_url=None)`…